imports/plugins/included/search-mongo/server/publications/searchresults.js

```diff
@@ -151,5 +156,8 @@ Meteor.publish("SearchResults", function (collection, searchTerm, facets, maxRes
   if (!searchTerm) {
     return this.ready();
   }
+  if(collection === "products" && searchTerm.length < 3 ) {
```

Comment:
Nice job, but is it possible to combine the conditional statements on line 156 with this line?
something like if (!searchTerm || (collection === "products" && searchTerm.length <3)) ...
Will it still give us the same result?